### PR TITLE
Create application settings

### PR DESCRIPTION
**Pull request by RedMarshall37**

### CODE REVIEW

File: Equality/Views/Teams/TeamPage.xaml [hunk 1]
```diff
@@ -15,6 +15,26 @@
            d:Background="{StaticResource MaterialDesignPaper}"
            d:DataContext="{d:DesignInstance vm:TeamPageViewModel, IsDesignTimeCreatable=True}"
            mc:Ignorable="d">
+    <Page.Resources>
+        <Style BasedOn="{StaticResource MaterialDesignTabRadioButton}"
```
Comment (igorgaming) [inline Equality/Views/Teams/TeamPage.xaml:19]: I believe you define this style in 2+ places, so move this in resource dictionary for all application (`Themes/Buttons`)

**Review by igorgaming — CHANGES_REQUESTED** [review]

File: Equality/Services/ThemeService.cs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+using Microsoft.Win32;
+
+namespace Equality.Services
+{
+    public class ThemeService : IThemeService
+    {
+        private IThemeService.Theme _currentTheme { get; set; }
+
+        delegate void SyncTheme();
```
[the remark this answers is not in the document]
Comment (RedMarshall37) [reply Equality/Services/ThemeService.cs:23]: No. We need to remove this?

[the remark this answers is not in the document]
Comment (igorgaming) [reply Equality/Services/ThemeService.cs:23]: Sure

[the remark this answers is not in the document]
Comment (RedMarshall37) [reply Equality/Services/ThemeService.cs:23]: i am wrong. We use it here
image
And here
image

**Pull request merged**